Sources/StrongContractResponse/All/StringProtocol.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  File.swift
+//  
+//
+//  Created by lsd on 07/05/24.
+//
+
+import Foundation
+//
+//  File.swift
+//
+//
+//  Created by Leonardo Dabus on 04/07/21.
+//
+
+import Foundation
+import protocol Foundation.DataProtocol
+import protocol Foundation.ContiguousBytes
+import struct Foundation.Data
```

Comment:
Add a comment explaining why you are importing this way.  It seems redundant,  you already ilportedfoundation

Reply:
Vapor does the same. 
<img width="902" alt="image" src="https://github.com/scott-lydon/StrongContractClient/assets/8468147/9ccd2747-f581-47b9-bacc-8a8fdd7bc801">

Reply:
Its a stylistic shortcoming on their part.  I remember reading style guides advising against it for Swift.  With older systems there were worthwhile performance improvements by being specific.  However, there is no longer a noticeable performance improvement with modern machines.  It also puts ambiguity for future devs, they might think: "Does this team require me to specify exactly what part of the framework I want to use?" So it adds work for no good reason.  Also the tendency that happens when people do this, is that you find yourself with a long stupid list of specific parts of the framework, which is kind of silly and begins to defeat the point of a framework (ease of use).

 Please just import the whole framework/library when using it.  Do not use dot notation to specify which part of the framework you're using.

Reply:
Probably what happened was they had a older C developer that joined the project, learned Swift, and then kept doing the same conventions he learned 40 years ago.